dev_guide/event_notifications.adoc

```diff
@@ -0,0 +1,155 @@
+[[dev-guide-event-notifications]]
+= Event Notifications
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+For each of your projects, you can choose to receive email notifications about
+various xref:../dev_guide/events.adoc#dev-guide-events[events], including:
+
+* _Dead deployment_: A failed deployment in which no other deployment in the
+rollout history is able to remain active. The related `deploymentConfig` has no
```

Comment:
Should we use the term "Deployment Configuration" instead? The use of `deploymentConfig` looks specific to the way we specify deployment configuration data in JSON/YAML format, but that's an implementation detail. In other words, I recommend referring to the concept rather than our specific implementation, which is narrower.

Consider linking to the appropriate doc as well, such as: https://docs.openshift.org/latest/architecture/core_concepts/deployments.html#deployments-and-deployment-configurations (aside: on this page, we refer to a `DeploymentConfiguration` object, but it's defined as `kind: DeploymentConfig` - either we should use a space "Deployment Configuration" when referring to the concept, or "DeploymentConfig" when referring to the object type)